revpi_common.c

```diff
@@ -247,3 +247,14 @@ int set_kthread_prios(const struct kthread_prio *ktprios)
 	read_unlock(&tasklist_lock);
 	return ret;
 }
+
```

Comment:
accept

Reply:
not quite sure whether the lib/crc8.c is 100% fitting our case.
my idea was trying to move the crc function from IOComm.c to revpi_common.c.
So in the mio patch, I just use the old one, and do the refactory later

Reply:
Ok, so lets stick with our custom crc function for now. However in future we should check whether it can be replaced with the kernel library function.